### ISSUE TITLE
Unable to Connect With Bot Allen Repository 

### ISSUE THREAD

**Issue by Parambakshi7** [botallen/repository.botallen#335]
![image](https://user-images.githubusercontent.com/121741646/210160844-1b2754f3-0a33-4513-8481-ceb8a7e66e84.jpg)

**Comment by LogicalError007** [comment]
Use DNS, but even if it installs all the add-ons. Channels are not working right now.

**Comment by Mujtiba** [comment]
Same issue here

**Comment by Mujtiba** [comment]
Unable to connect botallen repository.

**Comment by MeetBelani** [comment]
same issue here

**Comment by technofreak24** [comment]
Colors Marathi SD, Sony Marathi SD are the channels found to be working with a bit of break in the stream on 1st Jan 2023 at 2:30 PM IST. I will try to check other channels.
Using the addon with old version Kodi Matrix 19.0 on firetv stick (3rd gen) on Jio 4G mobile hotspot.

**Comment by jamalfeb1** [comment]
Help guys, shows same error

**Comment by technofreak24** [comment]
Attaching some working channel photos taken just minutes back of Kodi+addon (this thread is also visible in photo)
![IMG_20230101_181751_062529](https://user-images.githubusercontent.com/121749085/210171416-e58c826e-0af3-424d-8f7c-b3f3c73e855f.jpg)

![IMG_20230101_182023_062556](https://user-images.githubusercontent.com/121749085/210171427-8e916594-b0a7-4af3-8a74-c98369983166.jpg)

**Comment by Mujtiba** [comment]
![Uploading Screenshot_2023-01-01-18-53-33-357_org.xbmc.kodi.jpg…]()

**Comment by Mujtiba** [comment]
Still showing same error

**Comment by fingersfun** [comment]
Same Issue..

**Comment by CSaiteja99** [comment]
Guys use your mobile hotspot and it will work

**Comment by adityapv** [comment]
Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)

**Comment by jamalfeb1** [comment]
Can anyone provide steps to install zip file

Thanks

**Comment by fingersfun** [comment]
I tried uninstalling and installing Kodi and BotAllen repo but still
getting same Error.

**Comment by Marlinruno** [comment]
He won't do that, bcs imroot deleted the repository containing in a zip file from his botallen website, so no one can use it, if anybody who had taken a copy if that zip file can use that repository, and or anybody had a backup of that zip file..

**Comment by fingersfun** [comment]
Yes I have a zip file backup in my tv and I am trying to install from that
zip file. But still I am getting the same error. 😥

**Comment by CSaiteja99** [comment]
Bro , use your mobile hotspot and connect to the hotspot from ur Tv wifi
and try to install . It will work

**Comment by adityapv** [comment]
Check this one: https://github.com/Geletinousamigo/plugin.video.jiotv

**Comment by Parambakshi7** [comment]
> Check this one: https://github.com/Geletinousamigo/plugin.video.jiotv

Thnx this worked well

**Comment by Mujtiba** [comment]
Thanks it works
But pvr client not working

**Comment by Parambakshi7** [comment]
> Thanks, everything working fine...

Still working??

**Comment by thegrimmhealer** [comment]
I'm unable to install the zip file from Kodi

**Comment by LogicalError007** [comment]
> I'm unable to install the zip file from Kodi

Directly download the zip from GitHub and install through zip or use VPN or DNS to install from repository.

**Comment by thegrimmhealer** [comment]
The zip is on the device it's just that Kodi cannot detect the file. This is happening only on yhr chromecast with google TV and not my other device with is MI box 4

**Comment by LogicalError007** [comment]
> The zip is on the device it's just that Kodi cannot detect the file. This is happening only on yhr chromecast with google TV and not my other device with is MI box 4

Use the VPN method and source the repository.

**Comment by Mujtiba** [comment]
Kodi showing update jio aap Advertisement.... 
How to solve this problem now.
